### ISSUE TITLE
Occupational history of putting fingers in other people's mouths

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10693]
**Preferred term label:**
Occupational history of putting fingers in other people's mouths

**Synonyms**
Medical or dental profession

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Herpetic whitlow

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001120
- label: Occupational history of putting fingers in other people's mouths
- parents: [Occupational exposure history (HP:0035010)](https://hpo.jax.org/app/browse/term/HP:0035010)
- pmid: 
- def: A history of exposure of fingers by putting them in other peoples' mouths occupationally, e.g., as a dentist or physician.
- comment: This is a risk factor for Herpetic whitlow.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10693
- synonyms:
